styles/main.css

```diff
@@ -51,23 +53,22 @@ ul {
     min-width: 0;
     max-width: 1024px;
     margin: auto;
-    padding: 2em 0em 2em 0em;
+    padding: 0 2em 0 2em;

```

Comment:
Incorrect order.  It starts with padding-top and works around clockwise, so the 2nd and 4th entries are left & right padding, respectively